core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.ts

```diff
@@ -410,6 +410,30 @@ export class BaseUser {
   getCurrentUrlWithoutParameters(): string {
     return this.page.url().split('?')[0];
   }
+
+  /**
+   * This function Waits for a specific text to appear in the body of the page.
+   * This function only waits for rendered text. It does not wait for text in iframes or text that is hidden or not rendered for some reason.
+   * The text matching is case-sensitive and space-sensitive, so it might not find the text if there are differences in case or white space.
```

Comment:
I'm surprised these lines aren't throwing lint errors -- aren't they too long? 

Why is prettier not running locally on your submitted code? Can you get this fixed in your development environment?